data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+[vmi_lifecycle_test]
+GINKGO_FOCUS=vmi_lifecycle_test.go
```

Comment:
Please help add brief description of what this conf file and its sections are used for.

Reply:
Referring to https://confluence.suse.com/pages/viewpage.action?spaceKey=~nanzhang&title=KubeVirt+QA+automation+test+design#KubeVirtQAautomationtestdesign-Presettestconfigs